Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
Do we really need it in Git?

Reply:
AFAIK there is no clear best practice here. Some other projects do add it, such as [bitnami](https://github.com/bitnami/charts/tree/main/bitnami) whilst others don't, such as [prometheus-community](https://github.com/prometheus-community/helm-charts/tree/main/charts). I don't really mind either way, so if there's a stronger preference not to - I can remove it.

Reply:
Check these two lines:
1. https://github.com/prometheus-community/helm-charts/blob/main/.github/CODEOWNERS#L12
2. https://github.com/prometheus-community/helm-charts/blob/main/.github/CODEOWNERS#L15

And guess what I think. 😃 

Let's keep it. 👍